client/utils/account-fees.tsx

```diff
@@ -74,24 +74,25 @@ const getStripeFeeSectionUrl = ( country: string ): string => {
 	);
 };
 
-const getFeeDescriptionString = ( fee: BaseFee ): string => {
+const getFeeDescriptionString = (
+	fee: BaseFee,
+	discountRate: number
```

Comment:
I like the solution, but I also find `discountRate` misleading: it's a kinda `discountBasedMultiplier`.

Reply:
After some thoughts, refactored to `discountBasedMultiplier` in https://github.com/Automattic/woocommerce-payments/pull/7787/commits/1083bd4a5ece63704ce39c3a3dbf02d18b5c4faf